### PR TITLE
docs: add az keyvault user guide

### PR DESCRIPTION
**Pull request by stijnmoreels**
Add user guide to integrate the Arcus secret store with Azure Key Vault integration to an application.

### CODE REVIEW

File: docs/preview/02-Guides/add-secret-store-with-keyvault-integration.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    }
+}
+```
+
+> Note that during the configuration of the secret store, you will be able to access the application configuration; in this case it is used to retrieve the URI where the Azure Key Vault is located.
+
+### 3. Inject the Arcus secret store in application
+Now that the Arcus secret store is added and configured to the application, the application can use it to retrieve it secrets. The Arcus secret store is accessible throughout the application via the `ISecretProvider` interface - combining all the configured secret providers.
+
+In the `OrderController`, inject this interface to retrieve the Azure Service Bus connection string.
```
Comment (fgheysels) [inline docs/preview/02-Guides/add-secret-store-with-keyvault-integration.md:106]:
```suggestion
In the `OrderController`, inject the `ISecretProvider` interface via the constructor.  The `ISecretProvider` will allow you to retrieve the Azure Service Bus connection string from the secret store.
```

**Review by fgheysels — APPROVED** [review]
Neat.  Very clear, this will certainly help people in getting to understand / learn how to use this!

File: docs/preview/02-Guides/add-secret-store-with-keyvault-integration.md (hunk 1 above)
Comment (stijnmoreels) [reply docs/preview/02-Guides/add-secret-store-with-keyvault-integration.md:106]: Is good! Thx. 👍

**Comment by stijnmoreels** [comment]
> Neat. Very clear, this will certainly help people in getting to understand / learn how to use this!

That would be great, indeed! ❤️ 
Thx!

**Pull request merged**